docs/reference/actions.md

```diff
@@ -36,11 +36,12 @@ An action is a connection between:
 
 ## Descriptors
 
-The `data-action` value `click->gallery#next` is called an _action descriptor_. In this descriptor:
+The `data-action` value `click->gallery#next:once` is called an _action descriptor_. In this descriptor:
```

Comment:
Let’s revert the change to this example and the addition to the list below. Most actions won’t have any options and I think it’s a little overwhelming to see it up front. (The Options section below is good 👍)